### PR TITLE
Add a longer explanation to rollout docstring

### PR DESCRIPTION
**Pull request by Rocamonde**
## Description

Improve the docstring of rollout to explain how it's different from generate_trajectories.

## Testing

Description of how you've tested your changes.

### CODE REVIEW

File: src/imitation/data/rollout.py [hunk 1]
```diff
@@ -610,6 +610,11 @@ def rollout(
 ) -> Sequence[types.TrajectoryWithRew]:
     """Generate policy rollouts.
 
+    This method is a wrapper to generate_trajectories. In addition, it
```
Comment (AdamGleave) [inline src/imitation/data/rollout.py:613]:
```suggestion
    This method is a wrapper of generate_trajectories that allows
```

File: src/imitation/data/rollout.py [hunk 2]
```diff
@@ -610,6 +610,11 @@ def rollout(
...
+    allows the user to replace the rewards and observations with the original
```
Comment (AdamGleave) [inline src/imitation/data/rollout.py:614]:
```suggestion
    the user to additionally replace the rewards and observations with the original
```

**Review by AdamGleave — APPROVED** [review]
LGTM, some minor suggested changes in wording.

(Please request review from me or someone else in future to avoid long delays!)

**Comment by AdamGleave** [comment]
Tests failing, @Rocamonde  maybe need to merge master back into this branch since it's quite old now?

**Comment by Rocamonde** [comment]
> Tests failing, @Rocamonde maybe need to merge master back into this branch since it's quite old now?

Just did, let's see what happens. This is only a change in the docstrings, so tests shouldn't fail if the CI is clean.

**Pull request merged**